src/main/java/com/enterprise/redcord/RedCordController.java

```diff
@@ -26,27 +30,35 @@
     @RequestMapping("/")
     public String index(Model model) {
         Message messageEntry = new Message();
+
+        logger.trace("Accessed index method in RedCordController.");
+
         model.addAttribute(messageEntry);
         return "start";
     }
-    
-    
+
     
     @RequestMapping("/saveTopic")
     public String saveTopic(Topic topic) {
+       logger.trace("Accessed savedTopic method in RedCordController.");
+
        try {
            topicService.save(topic);
-       }catch(Exception e){
+       } catch(Exception e) {
+           logger.error("Error in saveTopic method: " + e.getMessage());
```

Comment:
Great.  Logging is good here.  And you can remove the printStackTrace(), as you can get the same information (and then some) from the logger.